### ISSUE TITLE
Instagram Feed library archived (doesn't work)

### ISSUE THREAD

**Issue by sbgraphicdesign** [backdrop-contrib/instagram_feed#5]
I'm getting a "Failed to load resource: net::ERR_BLOCKED_BY_RESPONSE" error for each image. 

Looks like an issue with CORS and Google hosting, which is causing the same problems for Drupal Instagram Feed Block module users. Module is pretty useless at the moment!

**Comment by ghost** [comment]
It looks like the [library](https://github.com/jsanahuja/jquery.instagramFeed) this module uses is no longer working and has been archived. We'll either need to deprecate this module or find an alternative solution... No promises as to if/when I can do this.

**Comment by laryn** [comment]
Looks like Instagram is discontinuing access without a token, unfortunately. Here is a script that came up in brief research:

 - [Instafeed.js](https://instafeedjs.com) - v2 is in RC

**Comment by stpaultim** [comment]
Yes, I also just tried the module. I'm getting the feed, but images are not working. 

![image](https://user-images.githubusercontent.com/3144571/161486264-c1ef3392-5aa3-4a8f-87d1-7d741373b92f.png)

Interestingly, the image src url works if I enter it directly into the browser. 

![image](https://user-images.githubusercontent.com/3144571/161486379-699ce2bf-5ca2-4393-a04d-19bef27f09bf.png)

If the modules really doesn't work, we should remove it from BackdropCMS.org
